internal/command/jsonformat/change/change.go

```diff
@@ -83,3 +89,19 @@ func (change Change) forcesReplacement() string {
 func (change Change) indent(indent int) string {
 	return strings.Repeat("    ", indent)
 }
+
+// emptySymbol returns an empty string that is the same length as an action
+// symbol (eg. '  +', '+/-', ...). It is used to offset additional lines in
+// change renderer outputs alongside the indent function.
+func (change Change) emptySymbol() string {
+	return "   "
+}
```

Comment:
Maybe we should extend `format.DiffActionSymbol` to return this for `plans.NoOp`…?

Reply:
I've added this to the set of general refactors, I'll do at the end.

Changing it here and now will make merging the rest of the PRs more difficult because of merge conflicts.

FWIW, I was also hesistant to do this as I didn't want to change anything about the way the other renderer works. It currently would render a NoOp with "  ?", which might be something it's expecting to happen.